app/views/pages/about_eraav.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="about_page page container">
+	<h2>About ERA A<span class="plus">&plus;</span>V</h2>
```

Comment:
Got some weird indentation/spacing here? It's like 4+ spaces? 

Also some html files have ending newlines (or multiple) and others don't